src/main.py

```diff
@@ -12,7 +12,7 @@
 
 version = "0.1.5"
 description = """
-The Nowcasting API is still under development. It only returns fake results.
+The Nowcasting API is still under development. It only returns zeroes for now.
```

Comment:
or is it zeros?

Reply:
https://english.stackexchange.com/questions/3824/what-is-the-plural-form-of-zero

Reply:
apparently both is correct haha: https://en.wiktionary.org/wiki/zero#Noun

Complicated language. I don't mind, whichever one u prefer! 😊

Reply:
It's interesting to note that the Oxford English Dictionary's Oxford Dictionaries site's sole definition of zeroes is related to zero as a verb, e.g. "watch as he zeroes his sights on the target"; not as the plural of zero. It states that the plural of zero is zeros.

Reply:
lets go zeros please - now i have looked at it too long

Reply:
hahahahhahaha sure